code/game/objects/items/food/monkeycube.dm

```diff
@@ -82,7 +82,7 @@
 
 /obj/item/food/monkeycube/gorilla
 	name = "gorilla cube"
-	desc = "A Waffle Co. brand gorilla cube. Now with extra molecules!"
+	desc = "A Waffle Corp brand gorilla cube. Now with extra molecules!"
```

Comment:
Should these be corp shortenings be punctuated, ie `A Waffle Corp. brand gorilla cube.`?

Reply:
Period is pretty much arbitrary. It can go wherever. Where you want it used?

Reply:
I would've thought any case of `Waffle Corp` should have a period (`Waffle Corp.`)

Reply:
Ok I added the period everywhere that doesn't have another punctuation after it because "Waffle Corp.'s" looks like shit.